common/source/docs/common-sonar-L04xMTW-GL04xMT.rst

```diff
@@ -0,0 +1,42 @@
+.. _common-sonar-L04xMTW-GL04xMT:
+
+=======================
+Sonar L04xMTW / GL04xMT  
+=======================
+
+The L04xMTW / GL04xMT is an echosounders (aka underwater sonar) with a 2 / 6 m range. More details can be found in the `datasheet <https://github.com/AlksSAV/Sonar-to-i2c-NMEA-/blob/main/L04xMTW%20%20GL04xMT%20datasheet.pdf>`__ 
+
+
+Supported sonar models:
+
+- L041MTW, GL041MT (max 6m range)
+- L042MTW, GL042MT (max 2m range)
+
+Recommended Hardware
+--------------------
+
+- L04xMTW / GL04xMT
+- Arduino nano
+
+The firmware file (sonar_to_nmea__SDDBT.ino) is located in this `repository <https://github.com/AlksSAV/Sonar-to-i2c-NMEA-/blob/main/sonar_to_nmea__SDDBT.ino>`__ 
+
+Connecting and Configuring
+--------------------------
+.. image:: ../../../images/Sonars nmea.png
```

Comment:
cant have whitespace in filename....also document not in a TOCtree....needs addition on rangefinder landing page TOCtree